runtime/rococo/src/lib.rs

```diff
@@ -208,14 +207,14 @@ construct_runtime! {
 		Hrmp: parachains_hrmp::{Pallet, Call, Storage, Event},
 		SessionInfo: parachains_session_info::{Pallet, Call, Storage},
 
+		// Parachain Onboarding Pallets
 		Registrar: paras_registrar::{Pallet, Call, Storage, Event<T>},
-		ParasSudoWrapper: paras_sudo_wrapper::{Pallet, Call},
```

Comment:
Do we need this still?

Offers:

* sudo_schedule_para_initialize
* sudo_schedule_para_cleanup
* sudo_queue_downward_xcm
* sudo_establish_hrmp_channel

Reply:
still useful for testing purpose, at least for rococo-local we want those

Reply:
ok, np